howto/application/stream-application.md

```diff
@@ -59,9 +61,31 @@ The downloaded `.csv` file has the following statistics:
 
 ### Sharing a streaming session
 
+
+````{tabs}
+
+```{group-tab} CLI
+
+You can share an authenticated session with another user by running:
+
+    anbox-stream-gateway share --description="Grant access to xxx"
```

Comment:
anbox-stream-gateway share --description="Grant access to xxx"
->
anbox-stream-gateway share <session_id> --description="Grant access to xxx"